tests/unittest/llmapi/test_executor.py

```
@@ -172,6 +172,62 @@ def test_invalid_sampling_params():
+    assert sp.stream_interval is None
+
+    # Invalid values
+    with pytest.raises(ValueError, match="stream_interval"):
+        SamplingParams(stream_interval=0)
+    with pytest.raises(ValueError, match="stream_interval"):
+        SamplingParams(stream_interval=-1)
+
+
+def test_stream_interval_openai_protocol():
+    from tensorrt_llm.serve.openai_protocol import (ChatCompletionRequest,
+                                                     CompletionRequest)
```

Flake8 (7.3.0)

[error] continuation line over-indented for visual indent

(E127)